packages/base-styles/_variables.scss

```diff
@@ -104,3 +104,16 @@ $block-selected-to-content: $block-edge-to-content - $block-selected-margin - $b
 $radius-round-rectangle: 4px;
 $radius-round: 50%;
 $radius-block-ui: 2px;
+
+:root {
```

Comment:
This chunk of code is going to be duplicated in all of our stylesheets. I considered doing it only on `edit-*` stylesheets but the downside is that the npm users won't have a default color as faallback.

Reply:
Do you mean this is needed for the `postcss-custom-properties` step? You could pass in the default scheme to the [postcss config](https://github.com/postcss/postcss-custom-properties#importfrom) — it would need a little mapping to get it into var format, but it should work.

Reply:
I updated now, and there's only the default one that's included so it should be fine and similar to the postcss config.